src/Raven.Server/Documents/Patch/ScriptRunner.cs

```diff
@@ -208,6 +211,10 @@ public SingleRun(DocumentDatabase database, RavenConfiguration configuration, Sc
 
                 ScriptEngine.SetValue("scalarToRawString", new ClrFunctionInstance(ScriptEngine, "scalarToRawString", ScalarToRawString));
 
+                //TimeSeries
+                ScriptEngine.SetValue("appendTs", new ClrFunctionInstance(ScriptEngine, "appendTs", AppendTimeSeries));
```

Comment:
`timeseriesAppend` - same for the rest

Reply:
This will not match e.g. counter methods where you have `incrementCounter`

Reply:
@ayende @ppekrol What should it be?

Reply:
Go with @ppekrol suggestion